modules/azure/src/main/java/org/testcontainers/containers/Constants.java

```diff
@@ -0,0 +1,24 @@
+package org.testcontainers.containers;
+
+/**
+ * @author Onur Kagan Ozcan
+ */
+final class Constants {
```

Comment:
Don't use a dedicated class for constants, rather move it into the appropriate class (`CosmosDBEmulatorContainer` and `KeyStoreUtils`?)

Reply:
👍 Added to my patch list.